core/trino-main/src/test/java/io/trino/sql/analyzer/TestAnalyzer.java

```diff
@@ -892,6 +892,26 @@ public void testWindowClause()
                 .hasErrorCode(DUPLICATE_WINDOW_NAME);
     }
 
+    @Test
+    public void testWindowNames()
```

Comment:
Add some tests for window name resolution that involve SQL identifier canonicalization semantics.

Reply:
This method checks how identifiers are compared (uppercase vs lowercase, non-delimited vs delimited). What is missing?

Reply:
I was thinking of tests like:

```
SELECT xxx() OVER w FROM t WINDOW W AS ...

SELECT xxx() OVER "w" FROM t WINDOW W AS ...

SELECT xxx() OVER w FROM t WINDOW "w" AS ...
```

and some including existing window reference names mixing delimited/undelimited identifiers.

Reply:
You mean analyzer tests involving named window references or query tests?

Reply:
Either would be fine